stepcvt/project.py

```diff
@@ -255,6 +256,7 @@ def from_dict(cls, d):
             raise ValueError(f"Incorrect value for type, expected CountInfo")
 
         return cls(count=d["count"])
+'''
 
```

Comment:
Don't use a multi-line string for this. Use comments, or remove it entirely since it will be available in version control.